BUILD.bazel

```diff
@@ -244,6 +247,7 @@ orfs_flow(
             "CORE_UTILIZATION": "20",
             "MACRO_PLACE_HALO": "20 20",
             "RTLMP_FLOW": "1",
+	    "BLOCKS": boom_tile_blocks,
```

Comment:
Why do you try to set BLOCKS? This will only confuse ORSF Makefile.

bazel-orfs sets MACROS, which is used by the .tcl scripts and is set to $(BLOCKS) by default.